### PR TITLE
fixed music page css

### PR DESCRIPTION
**Pull request by jut008**

### CODE REVIEW

File: client/src/css/music.css [hunk 1]
```diff
@@ -14,169 +14,169 @@ body {
...
   }
+}
+.music-page {
+  margin-top: 8em; /* Adjust based on your navbar height */
+}
 
-  
-
-  
+.soundSelector {
```
Comment (Shuzhengz) [inline client/src/css/music.css:170]: White noise clips out of bonds and the items don't look centered

image

image

**Review by Shuzhengz — CHANGES_REQUESTED** [review]

**Comment by Shuzhengz** [comment]
the alarm boxes also clips out of bounds

image

**Comment by Shuzhengz** [comment]
image

**Comment by JingtingL** [comment]
Fixed Misalignment in alarm modal boxes

**Comment by JingtingL** [comment]
Fixed Journal page misalignment

**Review by Shuzhengz — APPROVED** [review]
lgtm

**Pull request merged**